documentation/book/faq.adoc

```diff
@@ -1,9 +1,52 @@
-[appendix]
-== Frequently Asked Questions
 
-=== Cluster Operator
+= Frequently Asked Questions
 
-==== Log contains warnings about failing to acquire lock
+== Cluster Operator
+
+=== Why does {ProductName} cluster admin privileges to install?
+
+When installing {ProductName} you need to have the ability to create Custom Resource Definitions (CRDs). 
+These are the resources that tell {PlatformProductName} about the {ProductName}-specific resources, such as `Kafka`, `KafkaConnect` and so on.
+Because CRDs are a cluster-scoped resource (rather than being scoped to a particular {PlatformProductName} namespace) typically cluster admin privileges are required to be able to do this.
```

Comment:
Reworded to:

> Because CRDs are a cluster-scoped resource rather than being scoped to a particular {PlatformProductName} namespace, they typically require cluster admin privileges to install.